Source/JavaScriptCore/wasm/WasmAirIRGeneratorBase.h

```diff
@@ -2536,7 +2539,27 @@ auto AirIRGeneratorBase<Derived, ExpressionType>::addArrayGet(uint32_t typeIndex
     // https://bugs.webkit.org/show_bug.cgi?id=245405
     emitCCall(&operationWasmArrayGet, getValue, instanceValue(), self().addConstant(Types::I32, typeIndex), arrayref, index);
 
-    self().emitCoerceFromI64(elementType, getValue, result);
+    self().emitCoerceFromI64(resultType, getValue, result);
```

Comment:
Changes in this file were rebased on top of the armv7 BBQ patch that landed recently. Only some small adjustments were needed.